src/browser.ts

```diff
@@ -64,12 +60,12 @@ class GitHubFileBrowser extends Widget {
     this.userName.node.title = 'Click to edit user/organization';
     this._browser.toolbar.addItem('user', this.userName);
     this.userName.name.changed.connect(this._onUserChanged, this);
-
+    this.gitHubBaseUrl = "https://github.com"
```

Comment:
Can we still define the default above as a constant? We can then use it as a fallback if the value out of the settings system is invalid (i.e., an empty string).